src/test/kotlin/io/reactivex/rxkotlin/BasicKotlinTests.kt

```diff
@@ -41,6 +42,20 @@ class BasicKotlinTests : KotlinTests() {
         verify(a, times(1)).received("Hello")
     }
 
+    @Test fun testOnError() {
+        class TestException : RuntimeException()
+        val list = ArrayList<Throwable>()
+        RxJavaPlugins.setErrorHandler { list.add(it) }
+        try {
+            Observable.error<Any>(TestException()).subscribeBy()
+            assertEquals(1, list.size)
+            assertTrue(list[0] is OnErrorNotImplementedException)
```

Comment:
I see that you're using twice the same object, what about extracting to variables or use Kotlin built function like `with(list[0])`